Traceback (most recent call last):
  File "/home/conda/users/envs/users-pangeo/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/conda/users/envs/users-pangeo/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/conda/users/envs/users-pangeo/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/home/conda/users/envs/users-pangeo/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/conda/users/envs/users-pangeo/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/conda/users/envs/users-pangeo/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/conda/users/envs/users-pangeo/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
%run ../../../environment_set_up/Start_Dask_Cluster_Nebari.ipynb

------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mGatewayServerError[0m                        Traceback (most recent call last)
Cell [0;32mIn[6], line 1[0m
[0;32m----> 1[0m [43mget_ipython[49m[43m([49m[43m)[49m[38;5;241;43m.[39;49m[43mrun_line_magic[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mrun[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m[38;5;124;43m'[39;49m[38;5;124;43m../../../environment_set_up/Start_Dask_Cluster_Nebari.ipynb[39;49m[38;5;124;43m'[39;49m[43m)[49m

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/IPython/core/interactiveshell.py:2417[0m, in [0;36mInteractiveShell.run_line_magic[0;34m(self, magic_name, line, _stack_depth)[0m
[1;32m   2415[0m     kwargs[[38;5;124m'[39m[38;5;124mlocal_ns[39m[38;5;124m'[39m] [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mget_local_scope(stack_depth)
[1;32m   2416[0m [38;5;28;01mwith[39;00m [38;5;28mself[39m[38;5;241m.[39mbuiltin_trap:
[0;32m-> 2417[0m     result [38;5;241m=[39m [43mfn[49m[43m([49m[38;5;241;43m*[39;49m[43margs[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m
[1;32m   2419[0m [38;5;66;03m# The code below prevents the output from being displayed[39;00m
[1;32m   2420[0m [38;5;66;03m# when using magics with decodator @output_can_be_silenced[39;00m
[1;32m   2421[0m [38;5;66;03m# when the last Python token in the expression is a ';'.[39;00m
[1;32m   2422[0m [38;5;28;01mif[39;00m [38;5;28mgetattr[39m(fn, magic[38;5;241m.[39mMAGIC_OUTPUT_CAN_BE_SILENCED, [38;5;28;01mFalse[39;00m):

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/IPython/core/magics/execution.py:722[0m, in [0;36mExecutionMagics.run[0;34m(self, parameter_s, runner, file_finder)[0m
[1;32m    720[0m     [38;5;28;01mwith[39;00m preserve_keys([38;5;28mself[39m[38;5;241m.[39mshell[38;5;241m.[39muser_ns, [38;5;124m'[39m[38;5;124m__file__[39m[38;5;124m'[39m):
[1;32m    721[0m         [38;5;28mself[39m[38;5;241m.[39mshell[38;5;241m.[39muser_ns[[38;5;124m'[39m[38;5;124m__file__[39m[38;5;124m'[39m] [38;5;241m=[39m filename
[0;32m--> 722[0m         [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mshell[49m[38;5;241;43m.[39;49m[43msafe_execfile_ipy[49m[43m([49m[43mfilename[49m[43m,[49m[43m [49m[43mraise_exceptions[49m[38;5;241;43m=[39;49m[38;5;28;43;01mTrue[39;49;00m[43m)[49m
[1;32m    723[0m     [38;5;28;01mreturn[39;00m
[1;32m    725[0m [38;5;66;03m# Control the response to exit() calls made by the script being run[39;00m

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/IPython/core/interactiveshell.py:2939[0m, in [0;36mInteractiveShell.safe_execfile_ipy[0;34m(self, fname, shell_futures, raise_exceptions)[0m
[1;32m   2937[0m result [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mrun_cell(cell, silent[38;5;241m=[39m[38;5;28;01mTrue[39;00m, shell_futures[38;5;241m=[39mshell_futures)
[1;32m   2938[0m [38;5;28;01mif[39;00m raise_exceptions:
[0;32m-> 2939[0m     [43mresult[49m[38;5;241;43m.[39;49m[43mraise_error[49m[43m([49m[43m)[49m
[1;32m   2940[0m [38;5;28;01melif[39;00m [38;5;129;01mnot[39;00m result[38;5;241m.[39msuccess:
[1;32m   2941[0m     [38;5;28;01mbreak[39;00m

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/IPython/core/interactiveshell.py:268[0m, in [0;36mExecutionResult.raise_error[0;34m(self)[0m
[1;32m    266[0m     [38;5;28;01mraise[39;00m [38;5;28mself[39m[38;5;241m.[39merror_before_exec
[1;32m    267[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39merror_in_exec [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[0;32m--> 268[0m     [38;5;28;01mraise[39;00m [38;5;28mself[39m[38;5;241m.[39merror_in_exec

    [0;31m[... skipping hidden 1 frame][0m

File [0;32m/tmp/ipykernel_1027/3952966515.py:25[0m
[1;32m     23[0m         _env_to_add[_e] [38;5;241m=[39m os[38;5;241m.[39menviron[_e]
[1;32m     24[0m _options[38;5;241m.[39menvironment_vars [38;5;241m=[39m _env_to_add    
[0;32m---> 25[0m cluster [38;5;241m=[39m [43mgateway[49m[38;5;241;43m.[39;49m[43mnew_cluster[49m[43m([49m[43m_options[49m[43m)[49m          [38;5;66;03m##<< create cluster via the dask gateway[39;00m
[1;32m     26[0m cluster[38;5;241m.[39madapt(minimum[38;5;241m=[39m[38;5;241m2[39m, maximum[38;5;241m=[39m[38;5;241m30[39m)             [38;5;66;03m##<< Sets scaling parameters. [39;00m
[1;32m     28[0m client [38;5;241m=[39m cluster[38;5;241m.[39mget_client()

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:640[0m, in [0;36mGateway.new_cluster[0;34m(self, cluster_options, shutdown_on_close, **kwargs)[0m
[1;32m    617[0m [38;5;28;01mdef[39;00m [38;5;21mnew_cluster[39m([38;5;28mself[39m, cluster_options[38;5;241m=[39m[38;5;28;01mNone[39;00m, shutdown_on_close[38;5;241m=[39m[38;5;28;01mTrue[39;00m, [38;5;241m*[39m[38;5;241m*[39mkwargs):
[1;32m    618[0m [38;5;250m    [39m[38;5;124;03m"""Submit a new cluster to the gateway, and wait for it to be started.[39;00m
[1;32m    619[0m 
[1;32m    620[0m [38;5;124;03m    Same as calling ``submit`` and ``connect`` in one go.[39;00m
[0;32m   (...)[0m
[1;32m    638[0m [38;5;124;03m    cluster : GatewayCluster[39;00m
[1;32m    639[0m [38;5;124;03m    """[39;00m
[0;32m--> 640[0m     [38;5;28;01mreturn[39;00m [43mGatewayCluster[49m[43m([49m
[1;32m    641[0m [43m        [49m[43maddress[49m[38;5;241;43m=[39;49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43maddress[49m[43m,[49m
[1;32m    642[0m [43m        [49m[43mproxy_address[49m[38;5;241;43m=[39;49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mproxy_address[49m[43m,[49m
[1;32m    643[0m [43m        [49m[43mpublic_address[49m[38;5;241;43m=[39;49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_public_address[49m[43m,[49m
[1;32m    644[0m [43m        [49m[43mauth[49m[38;5;241;43m=[39;49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mauth[49m[43m,[49m
[1;32m    645[0m [43m        [49m[43masynchronous[49m[38;5;241;43m=[39;49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43masynchronous[49m[43m,[49m
[1;32m    646[0m [43m        [49m[43mloop[49m[38;5;241;43m=[39;49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mloop[49m[43m,[49m
[1;32m    647[0m [43m        [49m[43mcluster_options[49m[38;5;241;43m=[39;49m[43mcluster_options[49m[43m,[49m
[1;32m    648[0m [43m        [49m[43mshutdown_on_close[49m[38;5;241;43m=[39;49m[43mshutdown_on_close[49m[43m,[49m
[1;32m    649[0m [43m        [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m,[49m
[1;32m    650[0m [43m    [49m[43m)[49m

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:815[0m, in [0;36mGatewayCluster.__init__[0;34m(self, address, proxy_address, public_address, auth, cluster_options, shutdown_on_close, asynchronous, loop, **kwargs)[0m
[1;32m    803[0m [38;5;28;01mdef[39;00m [38;5;21m__init__[39m(
[1;32m    804[0m     [38;5;28mself[39m,
[1;32m    805[0m     address[38;5;241m=[39m[38;5;28;01mNone[39;00m,
[0;32m   (...)[0m
[1;32m    813[0m     [38;5;241m*[39m[38;5;241m*[39mkwargs,
[1;32m    814[0m ):
[0;32m--> 815[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_init_internal[49m[43m([49m
[1;32m    816[0m [43m        [49m[43maddress[49m[38;5;241;43m=[39;49m[43maddress[49m[43m,[49m
[1;32m    817[0m [43m        [49m[43mproxy_address[49m[38;5;241;43m=[39;49m[43mproxy_address[49m[43m,[49m
[1;32m    818[0m [43m        [49m[43mpublic_address[49m[38;5;241;43m=[39;49m[43mpublic_address[49m[43m,[49m
[1;32m    819[0m [43m        [49m[43mauth[49m[38;5;241;43m=[39;49m[43mauth[49m[43m,[49m
[1;32m    820[0m [43m        [49m[43mcluster_options[49m[38;5;241;43m=[39;49m[43mcluster_options[49m[43m,[49m
[1;32m    821[0m [43m        [49m[43mcluster_kwargs[49m[38;5;241;43m=[39;49m[43mkwargs[49m[43m,[49m
[1;32m    822[0m [43m        [49m[43mshutdown_on_close[49m[38;5;241;43m=[39;49m[43mshutdown_on_close[49m[43m,[49m
[1;32m    823[0m [43m        [49m[43masynchronous[49m[38;5;241;43m=[39;49m[43masynchronous[49m[43m,[49m
[1;32m    824[0m [43m        [49m[43mloop[49m[38;5;241;43m=[39;49m[43mloop[49m[43m,[49m
[1;32m    825[0m [43m    [49m[43m)[49m

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:920[0m, in [0;36mGatewayCluster._init_internal[0;34m(self, address, proxy_address, public_address, auth, cluster_options, cluster_kwargs, shutdown_on_close, asynchronous, loop, name)[0m
[1;32m    918[0m     [38;5;28mself[39m[38;5;241m.[39mstatus [38;5;241m=[39m [38;5;124m"[39m[38;5;124mstarting[39m[38;5;124m"[39m
[1;32m    919[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39masynchronous:
[0;32m--> 920[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mgateway[49m[38;5;241;43m.[39;49m[43msync[49m[43m([49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_start_internal[49m[43m)[49m

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:344[0m, in [0;36mGateway.sync[0;34m(self, func, *args, **kwargs)[0m
[1;32m    340[0m future [38;5;241m=[39m asyncio[38;5;241m.[39mrun_coroutine_threadsafe(
[1;32m    341[0m     func([38;5;241m*[39margs, [38;5;241m*[39m[38;5;241m*[39mkwargs), [38;5;28mself[39m[38;5;241m.[39mloop[38;5;241m.[39masyncio_loop
[1;32m    342[0m )
[1;32m    343[0m [38;5;28;01mtry[39;00m:
[0;32m--> 344[0m     [38;5;28;01mreturn[39;00m [43mfuture[49m[38;5;241;43m.[39;49m[43mresult[49m[43m([49m[43m)[49m
[1;32m    345[0m [38;5;28;01mexcept[39;00m [38;5;167;01mBaseException[39;00m:
[1;32m    346[0m     future[38;5;241m.[39mcancel()

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/concurrent/futures/_base.py:458[0m, in [0;36mFuture.result[0;34m(self, timeout)[0m
[1;32m    456[0m     [38;5;28;01mraise[39;00m CancelledError()
[1;32m    457[0m [38;5;28;01melif[39;00m [38;5;28mself[39m[38;5;241m.[39m_state [38;5;241m==[39m FINISHED:
[0;32m--> 458[0m     [38;5;28;01mreturn[39;00m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m__get_result[49m[43m([49m[43m)[49m
[1;32m    459[0m [38;5;28;01melse[39;00m:
[1;32m    460[0m     [38;5;28;01mraise[39;00m [38;5;167;01mTimeoutError[39;00m()

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/concurrent/futures/_base.py:403[0m, in [0;36mFuture.__get_result[0;34m(self)[0m
[1;32m    401[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39m_exception:
[1;32m    402[0m     [38;5;28;01mtry[39;00m:
[0;32m--> 403[0m         [38;5;28;01mraise[39;00m [38;5;28mself[39m[38;5;241m.[39m_exception
[1;32m    404[0m     [38;5;28;01mfinally[39;00m:
[1;32m    405[0m         [38;5;66;03m# Break a reference cycle with the exception in self._exception[39;00m
[1;32m    406[0m         [38;5;28mself[39m [38;5;241m=[39m [38;5;28;01mNone[39;00m

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:934[0m, in [0;36mGatewayCluster._start_internal[0;34m(self)[0m
[1;32m    932[0m     [38;5;28mself[39m[38;5;241m.[39m_start_task [38;5;241m=[39m asyncio[38;5;241m.[39mensure_future([38;5;28mself[39m[38;5;241m.[39m_start_async())
[1;32m    933[0m [38;5;28;01mtry[39;00m:
[0;32m--> 934[0m     [38;5;28;01mawait[39;00m [38;5;28mself[39m[38;5;241m.[39m_start_task
[1;32m    935[0m [38;5;28;01mexcept[39;00m [38;5;167;01mBaseException[39;00m:
[1;32m    936[0m     [38;5;66;03m# On exception, cleanup[39;00m
[1;32m    937[0m     [38;5;28;01mawait[39;00m [38;5;28mself[39m[38;5;241m.[39m_stop_internal()

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:947[0m, in [0;36mGatewayCluster._start_async[0;34m(self)[0m
[1;32m    945[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mstatus [38;5;241m==[39m [38;5;124m"[39m[38;5;124mcreated[39m[38;5;124m"[39m:
[1;32m    946[0m     [38;5;28mself[39m[38;5;241m.[39mstatus [38;5;241m=[39m [38;5;124m"[39m[38;5;124mstarting[39m[38;5;124m"[39m
[0;32m--> 947[0m     [38;5;28mself[39m[38;5;241m.[39mname [38;5;241m=[39m [38;5;28;01mawait[39;00m [38;5;28mself[39m[38;5;241m.[39mgateway[38;5;241m.[39m_submit(
[1;32m    948[0m         cluster_options[38;5;241m=[39m[38;5;28mself[39m[38;5;241m.[39m_cluster_options, [38;5;241m*[39m[38;5;241m*[39m[38;5;28mself[39m[38;5;241m.[39m_cluster_kwargs
[1;32m    949[0m     )
[1;32m    950[0m [38;5;66;03m# Connect to cluster[39;00m
[1;32m    951[0m [38;5;28;01mtry[39;00m:

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:536[0m, in [0;36mGateway._submit[0;34m(self, cluster_options, **kwargs)[0m
[1;32m    534[0m     options [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39m_config_cluster_options()
[1;32m    535[0m     options[38;5;241m.[39mupdate(kwargs)
[0;32m--> 536[0m resp [38;5;241m=[39m [38;5;28;01mawait[39;00m [38;5;28mself[39m[38;5;241m.[39m_request([38;5;124m"[39m[38;5;124mPOST[39m[38;5;124m"[39m, url, json[38;5;241m=[39m{[38;5;124m"[39m[38;5;124mcluster_options[39m[38;5;124m"[39m: options})
[1;32m    537[0m data [38;5;241m=[39m [38;5;28;01mawait[39;00m resp[38;5;241m.[39mjson()
[1;32m    538[0m [38;5;28;01mreturn[39;00m data[[38;5;124m"[39m[38;5;124mname[39m[38;5;124m"[39m]

File [0;32m/home/conda/users/d4b2efd231667a8c29119860ea8c7a8a71e87eba91be688f428dced45bfcda53-20230724-112151-738794-223-pangeo/lib/python3.10/site-packages/dask_gateway/client.py:418[0m, in [0;36mGateway._request[0;34m(self, method, url, json)[0m
[1;32m    416[0m     [38;5;28;01mraise[39;00m GatewayClusterError(msg)
[1;32m    417[0m [38;5;28;01melif[39;00m resp[38;5;241m.[39mstatus [38;5;241m==[39m [38;5;241m500[39m:
[0;32m--> 418[0m     [38;5;28;01mraise[39;00m GatewayServerError(msg)
[1;32m    419[0m [38;5;28;01melse[39;00m:
[1;32m    420[0m     resp[38;5;241m.[39mraise_for_status()

[0;31mGatewayServerError[0m: 500 Internal Server Error

Server got itself in trouble

